src/utils/appLinking.js

```diff
@@ -0,0 +1,117 @@
+import { Linking } from "react-native";
+import { Alert } from "../components/general/Alert";
+
+/**
+ * Object that holds the slack team id and several channel ids.
+ * The information in here is the most current as of May 28th, 2020.
+ * to find the team id or channel id login in slack on a computer.
+ * The url will look like this https://app.slack.com/client/{team_id}/{channel_id}
+ */
+
+export const slackInfo = {
+
+	team: "TC61JSPUZ",
+	channel: {
+		general: "CPRDXHHD4",
+		announcements: "CC5S86GHE",
+		fundraisingCommitte: "CCUUWRU1Y",
+		motorshpe: "CP5HC76TD",
+		projects: "CNATZRD9C",
+		shpejr: "GCWFE5630"
+	}
+};
+
+/**
+ * @description This function handles the action launching an app, like a mail app, from our app.
+ *
+ *  ** It requires to pass an url and optional warning message.**
```

Comment:
I don't believe you need this sentence since this information is already displayed by the params online 31 and 32